lib/config/options/index.ts

```diff
@@ -2093,6 +2093,13 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'expandCodeownersGroups',
+    description:
+      'Expand the configured code owner groups into a full list of group members.',
+    type: 'boolean',
+    default: false,
```

Comment:
add platform property with only gitlab, as this is only supported on gitlab

Reply:
`supportPlatforms`